EmerisHQ/emeris-libraries
# feat: backup mnemonic slideout

fixes: #56 

Currently it is implemented s.t. after the initial 'Back up Later' selection,
the next message is set to be after 1 hour.
Subsequent messages are set to display every 24 hours(when dismissed)

## Testing
- Create a new account
- Select the 'backup later' in the final stages of account creation
- Wait 1 hour and see the popup display.

In order to not wait 1 hour in order to test the feature
set the `60 * 60` in
https://github.com/EmerisHQ/emeris-libraries/blob/feat/backup-reminder/packages/browser-extension/src/views/AccountBackup.vue#L83
to `0`
